instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -84,7 +84,7 @@ def span_attrs(kind, *args) # rubocop:disable Metrics/AbcSize
             end
 
             attrs = { 'db.operation' => validated_operation(operation), 'db.postgresql.prepared_statement_name' => statement_name }
-            attrs['db.statement'] = sql unless config[:db_statement] == :omit
+            attrs['db.statement'] = sql.to_s if config[:db_statement] != :omit && sql&.to_s
```

Comment:
Nit, but: thoughts on changing to `attrs['db.statement'] = sql&.to_s if config[:db_statement] != :omit && !sql.nil?` or just `attrs['db.statement'] = sql&.to_s if config[:db_statement] != :omit`

Reply:
i've updated this to `attrs['db.statement'] = sql&.to_s unless config[:db_statement] == :omit`

I think this is as concise as i can get and handles all the edge cases and doesnt anger rubocop